ivre/utils.py

```diff
@@ -305,20 +305,50 @@ def get_domains(name):
     return ('.'.join(name[i:]) for i in range(len(name)))
 
 
+def espace_slash(string):
+    """This function transforms '\/' in '/'' if '\' was not escaped
+    """  # noqa: W605
+    escaping = False
+    new_string = ""
+    for char in string:
+        if not escaping and char == '\\':
+            escaping = True
+        elif escaping and char != '/':
+            new_string += '\\' + char
+            escaping = False
+        else:
+            new_string += char
+            escaping = False
+    return new_string
+
+
+def escape_first_slash(string):
```

Comment:
I don't get the purpose of this function.

Reply:
If one wants to search for the string `/test`, one would use `\/test` so that the first `/` does not make the string interpreted as a regular expression. Then, it is needed to remove this extra `\`.